docs/task-groups.md

```diff
@@ -0,0 +1,90 @@
+`relay task-groups`
```

Comment:
Before we land... do we want this to be `relay task-groups` or `relay task groups`? Similar question for `relay task-types`?

Reply:
Would you say the better course of action would be to 1) rename the directories for those from `task-types` to `task types` and similarly for `task-groups` to `task groups` or 2) move `types` and `groups` into being subdirectories within `tasks` and rename `tasks` directory to `task`?